jobrunner/service.py

```diff
@@ -28,7 +30,7 @@ def start_thread(target, name):
 def main():
     """Run the main run loop after starting the sync loop in a thread."""
     # extra space to align with other thread's "sync" label.
```

Comment:
I know it's only temporary, but can we update the docstring and comment to match the change?

Reply:
ah, yeah, good catch